integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,15 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
This line here would make this test be skipped when running with the LXC driver.
Since it seems like this should work just fine on LXC.... so can probably just remove the line.